packages/@tinacms/cli/src/buildTina/index.ts

```diff
@@ -185,6 +185,10 @@ export const buildCmdBuild = async (
     usingTs: ctx.usingTs,
     port: options.port,
   })
+  // @ts-ignore
+  if (ctx?.skipBuild) {
```

Comment:
Could this be maybe called "skipAdmin" or something more specific? My first thought was that this would skip the client build

Reply:
Updated